cmd/nrocli/main.go

```diff
@@ -0,0 +1,32 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/openshift-kni/numaresources-operator/pkg/commands"
+)
+
+func main() {
+	root := commands.NewRootCommand()
```

Comment:
I'm still unconvinced we need all this abstraction and generic infra just yet.

Reply:
Ok. I will change it